packages/app/src/components/pages/Home/useSort.js

```diff
@@ -29,12 +29,12 @@ const useSort = () => {
       let aDate = new Date(a[sortConfig.key])
       let bDate = new Date(b[sortConfig.key])
 
-      if (aDate < bDate) {
-        return sortConfig.direction === SORT_DIRECTION.UP ? 1 : -1
+      if (aDate > bDate) {
```

Comment:
What's the reason for this change? It seems like it should function exactly the same as the previous code.

Reply:
Claudia and I got feedback from the product team. They requested that we change the order that the submission date is sorted.

Reply:
Per our conversation, we'll change this back to the way it was.